wagtailmedia/contentstate.py

```diff
@@ -0,0 +1,64 @@
+from draftjs_exporter.dom import DOM
+from wagtail.admin.rich_text.converters.contentstate_models import Entity
+from wagtail.admin.rich_text.converters.html_to_contentstate import AtomicBlockEntityElementHandler
+
+from wagtailmedia.models import get_media_model
+
+
+def media_entity(props):
+    """
+    Helper to construct elements of the form
+    <embed embedtype="custommedia" id="1"/>
+    when converting from contentstate data
+    """
+    return DOM.create_element('embed', {
+        'embedtype': 'wagtailmedia',
+        'id': props.get('id'),
+        'title': props.get('title'),
+        'type': props.get('type'),
+
+        'thumbnail': props.get('thumbnail'),
+        'file': props.get('file'),
+
+        'autoplay': props.get('autoplay'),
+        'mute': props.get('mute'),
+        'loop': props.get('loop'),
```

Comment:
I wonder if it might be better to just serialize all the props as a single JSON-encoded attribute, rather than have to store them as separate HTML attributes like this. Then you wouldn’t have to do the annoying `True if attrs.get('autoplay') == 'true' else False` conversion back the other way around.